packages/discovery/src/peer-exchange/waku_peer_exchange_discovery.ts

```diff
@@ -236,6 +272,33 @@ export class PeerExchangeDiscovery
     this.queryingPeers.delete(peerIdStr);
     this.queryAttempts.delete(peerIdStr);
   }
+
+  private async checkPeerInfoDiff(
+    peerId: PeerId,
+    peerInfo: PeerInfo,
+    shardInfo?: ShardInfo
+  ): Promise<{ hasMaDiff: boolean; hasShardDiff: boolean }> {
```

Comment:
a bit hard to get what is `Ma` so I'd do `Maddr` or something
```suggestion
  ): Promise<{ hasMaddrDiff: boolean; hasShardDiff: boolean }> {
```